composer.json

```diff
@@ -21,11 +24,20 @@
         "oomphinc/composer-installers-extender": "^1.1 || ^2"
     },
     "config": {
+        "process-timeout": 0,
```

Comment:
Why are you attempting to disable the timeout? I think the default timeout is 5 minutes or more, if a package install takes longer than that it's fair to assume that something's wrong.

I'm also not sure this has any effect. Try changing it to `1`, everything still works fine, even though a few of the bigger packages take a few seconds to install.

Reply:
There was an issue in the past with the process taking longer than 5 minutes in some environments, but it was before my time, and I'm not sure if it would be a concern still. We could leave it out, but maybe @msherron has some thoughts.